cocos/renderer/gfx-agent/DeviceAgent.cpp

```diff
@@ -87,7 +87,12 @@ bool DeviceAgent::doInit(const DeviceInfo &info) {
     static_cast<CommandBufferAgent *>(_cmdBuff)->_queue = _queue;
     static_cast<CommandBufferAgent *>(_cmdBuff)->initAgent();
 
+    // arcore & arengine currently only supports gles, session update requires gl context 
+    #if USE_AR_MODULE && CC_PLATFORM == CC_PLATFORM_ANDROID
```

Comment:
Just move this to GFXDeviceManager.h, and disable the agent layer completely for AR purposes